### ISSUE TITLE
Score Sharing

### ISSUE THREAD

**Issue by BinaryBrain** [BinaryBrain/Elodie-Dream#34]
Sharing score with the world could be interesting in term of the game's lifetime.
Any ideas?
Maybe a webpage with every scores?

**Comment by BasileVu** [comment]
Yeah, I think something like the nth best scores for a level displayed on a webpage could be great indeed.

**Comment by jrabasco** [comment]
Good idea BUT : we should add more granularity to the scores, because right now you can just get like the perfect score or a bad one and if we do this you will see 20 time the same score at the top.

And after this we should care about security.

**Comment by Protectator** [comment]
I like the idea but this hides tons of work underneath.

**Comment by BinaryBrain** [comment]
Not that much. Sending HTTP requests with SFML is pretty easy. Indeed, securing it will be the hardest part, but it's still not very long to implement compared to the feature it add.

**Comment by jrabasco** [comment]
Yes of course, but I can't think of a really secure way without an SSL certificate.Plus, with the actual scoring system this is nearly meaningless as there is no way to have a wide range of possible scores, I think we should begin by adding more ways to score.
